daliuge-engine/test/manager/test_daemon.py

```diff
@@ -27,11 +27,11 @@
 import unittest
 
 from dlg import utils, restutils
-from dlg.manager import constants
+from dlg import constants
 from dlg.manager.client import MasterManagerClient
 from dlg.manager.proc_daemon import DlgDaemon
 
-_TIMEOUT = 5
+_TIMEOUT = 3
```

Comment:
3 is likely too low - I have success getting this to work locally with 10 second timeout (which I know you previously tried here).

Reply:
Yes, I tried 10s, but it did not change anything, except that the time you had to wait until the tests showed up as failed got incredibly long. The original value was 5s and I will change it back to that. There are actually two timeouts, one for the startup of the manager (this one) and another for the client to try to connect to the manager (10s). Usually that's the one that fails, but that is not the actual problem, the problem is that the manager does not start up for some reason. Locally I never have issues, since the startup is much faster than even 3s.